.circleci/config.yml

```diff
@@ -18,12 +18,16 @@ jobs:
     executor: linux
     steps:
       - checkout
-      - restore_cache:
```

Comment:
we should keep the npm cache otherwise we'll dramatically increase the runtime of the tests

Reply:
this is for testing only ll be reverting later on